nosql-db/mongodb/src/test/java/io/quarkus/ts/nosqldb/mongodb/MongoClientMtlsIT.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.ts.nosqldb.mongodb;
+
+import io.quarkus.test.bootstrap.MongoDbService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.Mount;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MongoClientMtlsIT extends AbstractMongoClientIT {
```

Comment:
Personally I keep forgetting to add it to my PRs, but since you named this PR "Add coverage for Quarkus-6233", I think you should add tag here for that one